test/cell_test.rb

```diff
@@ -37,4 +39,37 @@ def test_fired_upon?
     assert_equal true, @cell1.fired_upon?
   end
 
+  def test_health_fire_upon
+    @cell1.place_ship(@cruiser)
+    assert_equal 3, @cell1.ship.health
+
+    @cell1.fire_upon
+    assert_equal 2, @cell1.ship.health
+  end
+
+  def test_render         #Should we break this up?
```

Comment:
Unsure of best practice, should we break this up into multiple tests as it is all within the same method?

Reply:
I think we can keep it in one since it is testing the same the same method!